drivers/can/can_mcux_flexcan.c

```diff
@@ -1190,51 +1191,65 @@ static int mcux_flexcan_init(const struct device *dev)
 	k_sem_init(&data->tx_allocs_sem, MCUX_FLEXCAN_MAX_TX,
 		   MCUX_FLEXCAN_MAX_TX);
 
-	data->timing.sjw = config->sjw;
```

Comment:
We still need to store the devicetree configured SJW value in `data->timing` somewhere, as this is what is used as default for later calls to `can_set_timing()`. We could just move it from the `config` struct to the `data` struct in `FLEXCAN_DEVICE_INIT_MCUX()`.

Reply:
My bad. Fixed it [here](https://github.com/ZEISS/zephyr/blob/1d719ea69d12b45b30b68263e3d242138099090d/drivers/can/can_mcux_flexcan.c#L1505).